docs/docs/02-concepts/01-preflight-and-inflight.md

```diff
@@ -334,6 +334,83 @@ Within the first clause of the `lift` block, a list of qualifications on preflig
 Statements within a `lift` block are exempt from the compiler's analyzer that tries to determine preflight object usage automatically.
 If an inflight method is directly or indirectly called within a `lift` block without sufficient resource qualifications, it may result in errors at runtime.
 
+## Building liftable abstractions
+
+Compute environments where inflight code is executed are called **inflight hosts**.
+For example, AWS Lambda, Fly.io machines, and Fargate containers are all examples of compute environments that can run inflight code.
+
+In order to run a piece of inflight code, the inflight host may need to be set up in a certain way.
+For example, to run an inflight function that reads from a storage bucket, the inflight host may require permissions to read from the bucket, or may require certain environment variables to be set, or may require network policies must be configured.
```

Comment:
Think it's worth linking to https://www.winglang.io/docs/standard-library/cloud/bucket for `storage bucket`?

I'm a fan of helping people out when these terms may be new to them.

Reply:
Sure thing, done